src/DotNet/DotNet.csproj

```diff
@@ -157,7 +161,7 @@
       Outputs="$(DotNetPacksDirectory).stamp"
       DependsOnTargets="_ReadAllNuGetSources">
     <Exec
-        Command="&quot;$(DotNetToolPath)&quot; workload install %(_WorkloadIds.Identity) --skip-manifest-update --verbosity diag --temp-dir &quot;$(DotNetTempDirectory)&quot; @(_AllNuGetSources->'--source &quot;%(Identity)&quot;', ' ')"
```

Comment:
Can we remove the _AllNuGetSources from the csproj?

Reply:
Yes if everything is using `-configfile` now, we wouldn't need that either.

Reply:
we don't specify `-configfile` anywhere, it's picked up by default is that ok?

Reply:
Just to be safe, might be worth using `--configfile &quot;$(MauiRootDirectory)NuGet.config&quot;`.

The one that uses a temporary file could be `--configfile &quot;$(DotNetTempDirectory)NuGet.config&quot;`.

Reply:
ok fixed!